runtime/gc_verbose_handler_realtime/VerboseHandlerOutputRealtime.cpp

```diff
@@ -194,7 +194,7 @@ MM_VerboseHandlerOutputRealtime::handleInitializedInnerStanzas(J9HookInterface**
 void
 MM_VerboseHandlerOutputRealtime::handleEvent(MM_MetronomeIncrementStartEvent* eventData)
 {
-	/* if we are in a sync gc cycle, do nothing here and let the syncgc stanza handle things */
+	/* if we are in a sync gc cycle, do nothing here and let the sync gc stanza handle things */
```

Comment:
Dmitri specifically said this should remain as `syncgc`

Reply:
In https://github.com/eclipse/openj9/pull/5897#pullrequestreview-242318978

Reply:
I reverted this change.